src/Raven.Client/Exceptions/Database/IndexCompactionInProgressException.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Raven.Client.Exceptions.Database
+{
+    internal class IndexCompactionInProgressException : RavenException
```

Comment:
move to Raven.Client.Exceptions.Documents.Indexes

Reply:
Done